### PR TITLE
Add a sentence to installation page to link the "activate license" page so enterprise users understand where to go

### PR DESCRIPTION
**Pull request by 09jvilla**
Right now our installation docs mix instructions on how to install Grafana and how to install Grafana Enterprise. This makes sense since its basically the same process but just with a different image/binary. 

However none of the docs link to our "activate you're grafana enterprise license" page, which is a necessary step after you install the enterprise binary if you want to actually use Grafana Enterprise. 

Adding a one sentence change to call this out on the top level installation page.

### CODE REVIEW

File: docs/sources/installation/_index.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ This section discusses the hardware and software requirements as well as the pro
 - [Install on macOS]({{< relref "mac" >}})
 - [Install on Windows]({{< relref "windows" >}})
 - [Run Docker image]({{< relref "docker" >}})
+- [Deploy on Kubernetes]({{< relref "kubernetes" >}})
 
 Refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}) topic. If you need to restart Grafana, refer to the [Restart Grafana]({{< relref "Restart-grafana.md" >}})
 topic for detailed information.
+
+If you want to install Grafana Enterprise, follow the instructions for installing Grafana on your particular operating system and then refer to [Activate an Enterprise license]({{< relref "activate-license.md" >}}).
```
Comment (09jvilla) [inline docs/sources/installation/_index.md:24]: The relref I'm using here is broken :/ 
I'm trying to figure out how to point to https://grafana.com/docs/grafana/latest/enterprise/activate-license/
which is under filepath docs/sources/enterprise/activate-license.md
but no luck. 

Can you help me understand what filepath our system is looking at when it does the build?

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

File: docs/sources/installation/_index.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ This section discusses the hardware and software requirements as well as the pro
 - [Install on macOS]({{< relref "mac" >}})
 - [Install on Windows]({{< relref "windows" >}})
 - [Run Docker image]({{< relref "docker" >}})
+- [Deploy on Kubernetes]({{< relref "kubernetes" >}})
 
 Refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}) topic. If you need to restart Grafana, refer to the [Restart Grafana]({{< relref "Restart-grafana.md" >}})
 topic for detailed information.
+
+If you want to install Grafana Enterprise, follow the instructions for installing Grafana on your system of choice and then refer to [Activate an Enterprise license]({{< relref "../enterprise/activate-license.md" >}}).
```
Comment (osg-grafana) [inline docs/sources/installation/_index.md:24]:
```suggestion
To install Grafana Enterprise, refer to the instructions for installing Grafana on your system. Then, refer to [Activate an Enterprise license]({{< relref "../enterprise/activate-license.md" >}}).
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]

File: docs/sources/installation/_index.md (hunk 1 above)
Comment (osg-grafana) [reply docs/sources/installation/_index.md]: Is this unrelated to the merge conflict?
`
Auto-merging docs/sources/installation/_index.md
--
14 | CONFLICT (content): Merge conflict in docs/sources/installation/_index.md`

**Comment by 09jvilla** [comment]
Closing this and reopening a clean PR -- seems like I kicked off a much more wide ranging review when I had to rebase.

**Pull request closed without merging**